variables.tf

```diff
@@ -67,16 +67,44 @@ variable "encryption_password" {
   default     = ""
 }
 
-variable "external_services" {
+variable "prefix" {
   type        = "string"
-  description = "object store provider for external services. Allowed values: gcs"
-  default     = ""
+  description = "Resource and instance prefix"
+  default     = "tfe"
 }
 
-variable "install_type" {
+variable "primary_count" {
   type        = "string"
-  description = "Installation type, options are (poc or production). Switch to production for external services."
-  default     = "poc"
+  description = "Number of primary nodes to run, must be odd number - 3 or 5 recommended."
+  default     = "3"
+}
+
+variable "secondary_count" {
+  type        = "string"
+  description = "Number of secondary nodes to run"
+  default     = "0"
+}
+
+variable "region" {
+  type        = "string"
+  description = "The region to install into."
+  default     = "us-central1"
+}
+
+variable "zone" {
+  type        = "string"
+  description = "Preferred zone"
+  default     = "us-central1-a"
+}
+
+###################################################
+# Optional External Services Variables
+###################################################
+
+variable "external_services" {
```

Comment:
In AWS, we're detecting if they pass in the postgresql address to decide if we're doing external services implicitly. Do we want to do that here?

Reply:
AWS is the default value for this, which is why you can skip it. Azure and GCP must pass in the type of external storage.

Reply:
Turns out I wasn't even using this in the code. Removed it from the variables file.